templates/taxi/manufacturer_list.html

```diff
@@ -7,7 +8,10 @@ <h1>
       Create
     </a>
   </h1>
-
+  <form method="get" action="" class="form-inline">
```

Comment:
The form method is set to 'get', which is appropriate for search forms as it appends the search query to the URL.